### DISCUSSION TITLE
drop+v6

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8701] [General]
it works well in v5:
`this.canvas.on({ drop: this.drop })`
but it is wrong in v6.it seems drop hook doesnot work.
is there any changes about  drop event between v5 and v6?i didnot see any instructions.

**Comment by wuxingping** [comment]
> it works well in v5: `canvas.on({ drop: this.drop })` but it is wrong in v6.it seems drop hook doesnot work. is there any changes about drop event between v5 and v6?i didnot see any instructions.

i knew this problem.you must add monitor dragover event like below for canvas, and then the drop event would works well in fabirc v6(6.0.0-beta1):
`canvas.on("dragover":(e){e.e.preventDefault()})`
but you donot need to do like that in fabirc v5

**Reply by ShaMan123** [reply]
interesting
it is probably because of #7802

**Reply by ShaMan123** [reply]
Anyways it now conforms to HTML spec. You should do that to allow the drop

**Reply by asturur** [reply]
@wuxingping can you make a reproducible case of what does not work? I would like a simple use case `on('drop', dropFunction)` to work out of the box
